test/mdc2_internal_test.c

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2016 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/* Unit tests for the mdc2 module */
```

Comment:
It isn't disabled by default.  I thought it was and was surprised to discover I was wrong.